sys/include/net/gnrc/rpl.h

```diff
@@ -82,6 +82,17 @@
  *   CFLAGS += -DCONFIG_GNRC_RPL_WITHOUT_VALIDATION
  *   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
+ * TODO
+ * ------
+ *
+ * The GNRC RPL implementation only implements storing mode with OF0 ([RFC6552](https://tools.ietf.org/html/rfc6552)).
+ * For interoparability with other RPL interoperability, open task include:
+ *
+ * - IPv6 Hop-by-hop RPL option
```

Comment:
Currently in (slow) progress :-).

Reply:
Maybe add https://github.com/RIOT-OS/RIOT/pull/7231 or https://github.com/RIOT-OS/RIOT/pull/7231#issuecomment-651237343 as a reference here, so eager people don't start implementing when there is already a partly solution.